README.md

```diff
@@ -26,7 +26,7 @@ Check out the [`zipkin-server`](/zipkin-server) documentation for configuration
 ## Core Library
 The [core library](https://github.com/openzipkin/zipkin/tree/master/zipkin/src/main/java/io/zipkin) requires minimum language level 7. While currently only used by the server, we expect this library to be used in native instrumentation as well.
 
-This includes built-in codec for both thrift and json structs. Direct dependencies on thrift or moshi (json library) are avoided by minifying and repackaging classes used. The result is a 256k jar which won't conflict with any library you use.
```

Comment:
lie.. we actually crept up to 272K! I wrote in 190, not 188, to give a little room :)